core/templates/list.h

```diff
@@ -762,4 +764,35 @@ class List {
 	}
 };
 
+template <class T, class A>
+void List<T, A>::Element::transfer_to_back(List<T, A> *p_dst_list) {
+	// Detach from current.
+
+	if (data->first == this) {
+		data->first = data->first->next_ptr;
+	}
+	if (data->last == this) {
+		data->last = data->last->prev_ptr;
+	}
```

Comment:
Shouldn't we also update where `next_ptr`'s `prev_ptr` points and where `prev_ptr`'s `next_ptr` points?

Reply:
Very true. It's after all the same as `erase()` has to do. Fixed.